marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/smart-mastering/merging/merging-function-arguments.xqy

```diff
@@ -0,0 +1,168 @@
+          "function": "quickStartMergeProperties"
+        }
+    }
+  },
+  "merging": array-node {
+    object-node {
+      "propertyName":"property1",
+      "algorithmRef": "quickStartMergeProperties",
+      "maxValues": "1"
+    }
+  },
+  "tripleMerge": object-node {
+    "namespace": "http://marklogic.com/smart-mastering/xqueryMerging",
+    "at": "/test/suites/data-hub/5/smart-mastering/merging/test-data/xqueryMergingFunctions.xqy",
+    "function": "quickStartMergeTriples"
+  }
+};
+
+declare variable $hub-central-javascript-options := object-node {
+  "mergeRules": array-node {
+    object-node {
+      (: using documentXPath to avoid the need to load an entity  :)
+      "documentXPath":"/property",
+      "mergeModulePath": "/test/suites/data-hub/5/smart-mastering/merging/test-data/javascriptMergingFunctions.sjs",
+      "mergeModuleFunction": "hubCentralMergeProperties",
+      "maxValues": "1"
+    }
+  },
+  "tripleMerge": object-node {
+    "at": "/test/suites/data-hub/5/smart-mastering/merging/test-data/javascriptMergingFunctions.sjs",
+    "function": "hubCentralMergeTriples"
+  }
+};
+
+declare variable $hub-central-xquery-options := object-node {
+  "mergeRules": array-node {
+    object-node {
+      (: using documentXPath to avoid the need to load an entity  :)
+      "documentXPath":"/property",
+      "mergeModulePath": "/test/suites/data-hub/5/smart-mastering/merging/test-data/xqueryMergingFunctions.xqy",
+      "mergeModuleNamespace": "http://marklogic.com/smart-mastering/xqueryMerging",
+      "mergeModuleFunction": "hubCentralMergeProperties",
+      "maxValues": "1"
+    }
+  },
+  "tripleMerge": object-node {
+    "at": "/test/suites/data-hub/5/smart-mastering/merging/test-data/xqueryMergingFunctions.xqy",
+    "namespace": "http://marklogic.com/smart-mastering/xqueryMerging",
+    "function": "hubCentralMergeTriples"
+  }
+};
+
+declare variable $test-docs := (
+  document { object-node { "property": "val1" } },
+  document { object-node { "property": "val2" } }
+);
+
+(
+  (: Test XQuery triple functions :)
+  try {
+    merging-impl:build-final-triples(merging-impl:compile-merge-options($quickstart-xquery-options), $test-docs, ()),
+    test:assert-true(fn:true())
+  } catch ($e) {
+    test:assert-false(fn:true(), "Unexpected exception for XQuery QuickStart triple merge. Exception: " || xdmp:describe($e, (), ()))
+  },
+  try {
+    merging-impl:build-final-triples(merging-impl:compile-merge-options($hub-central-xquery-options), $test-docs, ()),
+    test:assert-true(fn:true())
+  } catch ($e) {
+    test:assert-false(fn:true(), "Unexpected exception for XQuery Hub Central triple merge. Exception: " || xdmp:describe($e, (), ()))
+  },
+  try {
+    merging-impl:get-merge-values(merging-impl:compile-merge-options($quickstart-xquery-options) => map:get("mergeRulesInfo"), $test-docs, map:map(), map:map(), "options-ref"),
```

Comment:
Same thing for get-merge-values - looking at the function signature, it's not clear what this function does or what it returns. Assertions on the output would both verify and document the functionality.